src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Use 'const' for 'price' since it is not reassigned later in the function, which ensures the value does not get changed unintentionally.